### DISCUSSION TITLE
Consider how uncertainty from simplification will impact the goals of transferability and comparability

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#189] [SCI feedback]
The standard is vague in many places which introduces uncertainty. For example, the section on Embodied Emissions permits the use of "simple models" without qualification what these are. The resulting uncertainty however will negatively affect comparability of any calculated SCI indicators between alternative software. 

I miss consideration of this problem in the text.

**Comment by atg-abhishek** [comment]
@dschien please feel free to create a PR that clarifies this and brings forward what you would like to communicate on this and in which part of the SCI
